canonicalwebteam/discourse_docs/app.py

```diff
@@ -0,0 +1,35 @@
+from canonicalwebteam.discourse_docs.blueprint import build_blueprint
+from canonicalwebteam.discourse_docs.models import DiscourseApi
+
+
+class DiscourseExtension(object):
```

Comment:
Do you think this needs the word "Extension" in the name? The example in http://flask.pocoo.org/docs/1.0/extensiondev/ doesn't have it (`SQLite3`). Could we simply call it `DiscourseDocs`? Or does that somehow clash with another object we've already named?